### PR TITLE
EVG-18821 Surface commit queue link 

### PR DESCRIPTION
**Pull request by khelif96**
EVG-18821

### Description
Adds commit queue link to nav bar and adds a project selector on the commit queue page

### Screenshots
image

### CODE REVIEW

File: src/pages/CommitQueue.tsx [hunk 1]
```diff
@@ -37,19 +39,32 @@ export const CommitQueue: React.VFC = () => {
...
+              <Badge variant="darkgray">
+                {buildBadgeString(queue ? queue.length : 0)}
+              </Badge>
+            }
+            loading={loading}
+            size="large"
+          />
+          {commitQueue?.message && (
+            <Body data-cy="commit-queue-message">{commitQueue.message}</Body>
+          )}
```
Comment (minnakt) [inline src/pages/CommitQueue.tsx:57]: I'm thinking that this should be removed from the `<PageHeader />` and instead be placed directly above the `<HR />`. Then you could also delete the `Column` style

File: src/pages/CommitQueue.tsx [hunk 2]
```diff
@@ -86,3 +101,18 @@ const buildBadgeString = (queueLength: number): string => {
   }
   return `${queueLength} Item`;
 };
+
+const PageHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
```
Comment (minnakt) [inline src/pages/CommitQueue.tsx:108]: (nit. not necessary to specify)

```suggestion

```

**Review by minnakt — APPROVED** [review]
any chance you could also fix those funky looking buttons in your screenshot? 👀 not necessary ofc but would be a nice improvement!

File: src/pages/CommitQueue.tsx (hunk 1 above)
Comment (khelif96) [reply src/pages/CommitQueue.tsx:57]: image
Done

**Pull request merged**